### DISCUSSION TITLE
Internal package not found when running Prisma seed script

### DISCUSSION THREAD

**Discussion by unclejustin** [vercel/turborepo#7523] [Help]
### Summary

I have a Prisma seed script that references an internal package. When I try to transpile with `tsc` or run directly with `tsx` it throws this error: "Cannot find module 'internal-package' or its corresponding type declarations."

Side note: VSCode does not complain about the package.

I set up the internal package as per the [internal package docs.](https://turbo.build/repo/docs/handbook/sharing-code/internal-packages) 

No tsconfig file. `package.json` set up like below.

Prisma seed tries to reference the package like `import { something } from 'internal-package'`

### Additional information

```js
{
  "name": "internal-package",
  "version": "0.0.0",
  "private": true,
  "exports": "./src/index.ts",
  "main": "./src/index.ts",
  "types": "./src/index.ts",
  "dependencies": {
  }
}
```

**Comment by anthonyshew** [comment]
Please provide a reproduction.
